core/dbt/events/functions.py

```diff
@@ -57,7 +57,11 @@ def setup_event_logger(log_path, level_override=None):
     EVENT_HISTORY = deque(maxlen=flags.EVENT_BUFFER_SIZE)  # type: ignore
 
     make_log_dir_if_missing(log_path)
-    this.format_json = flags.LOG_FORMAT == "json"
+    # Flags is not set in the integration test suite but is by the environment.
+    # DBT_LOG_FORMAT is used in our test suite so checking for it's presence is key.
```

Comment:
I'm a little confused by this comment.

Reply:
Well that's because it was a placeholder of sorts since the solution I originally had was a jackhammer for a nail. Revised!